### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by biklas7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+  store.
+- Create a [Service
+  Account](https://github.com/Triple-T/gradle-play-publisher/#service-account)
+  to use with [Gradle Play Publisher
+  plugin](https://github.com/Triple-T/gradle-play-publisher/).
+- Add your keystore passwords and the keystore and service account files to
+  [Semaphore Secrets](https://docs.semaphoreci.com/essentials/using-secrets/).
+
+## License
+Copyright (c) 2019 Rendered Text
```
Comment (tmpduarte) [inline README.md]:
```suggestion
Copyright (c) 2020 Rendered Text
```

**Review by tmpduarte — DISMISSED** [review]
@biklas7 please just check the copyright date. I don't know which year should we use

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
-# semaphore-demo-android
\ No newline at end of file
+[![Build
+Status](https://semaphore-demos.semaphoreci.com/badges/semaphore-demo-android/branches/master.svg)](https://semaphore-demos.semaphoreci.com/projects/semaphore-demo-android)
+
+# Semaphore demo CI/CD pipeline for Android
+This is an example project and CI/CD pipeline showing how to build, test, and
+deploy a Android application using Semaphore 2.0.
```
Comment (tmpduarte) [inline README.md]:
```suggestion
deploy an Android application using Semaphore 2.0.
```

**Comment by biklas7** [comment]
> @biklas7 please just check the copyright date. I don't know which year should we use

@tmpduarte i think I've copied that from another demo project 🤔

**Review by tmpduarte — APPROVED** [review]

**Pull request merged**